content/500-support/100-help-articles/500-comparing-columns-through-raw-queries.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Comparing columns of same table through raw queries'
+metaTitle: 'Comparing columns of same table through raw queries'
+metaDescription: 'Comparing columns of same table through raw queries'
+---
+
+## Issue
+
+Comparing different columns from the same table is a common scenario that developers encounter. Some examples include comparing two numeric values in the same table or comparing two dates in a same table.
+
+
+## Solution
```

Comment:
As mentioned somewhere else, not a "Solution" but a "Workaround".

Reply:
Updated the heading from "Solution" to "Workaround"